internal/engine/eval/trusty/config.go

```diff
@@ -49,15 +49,19 @@ type ecosystemConfig struct {
 	Activity float64 `json:"activity" mapstructure:"activity"`
 }
 
-// config is the configuration for the vulncheck evaluator
+// config is the configuration for the trusty evaluator
 type config struct {
 	Action          pr_actions.Action `json:"action" mapstructure:"action" validate:"required"`
+	Block           map[string]bool   `json:"block" mapstructure:"block"`
 	EcosystemConfig []ecosystemConfig `json:"ecosystem_config" mapstructure:"ecosystem_config" validate:"required"`
 }
 
 func defaultConfig() *config {
 	return &config{
-		Action: pr_actions.ActionSummary,
+		Action: pr_actions.ActionReviewPr,
+		Block: map[string]bool{
+			"malicious": true,
```

Comment:
Isn't it safer if we invert this and don't rely on initialising the bool with true? 

Also I think we can even delete this option in the Review action. I mean turning it on kind of overlaps with the other actions and if someone doesn't want Minder to block the PR, they can use some of them, right? 

I think this is how the vulnerability/OSV rule works like, right?

Reply:
Ah so the idea is that this option is _for_ the review action. In addition to malicious, we will have another entry to block (or not) when deprecated packages are found. This will let users control when they want to block or just get feedback from minder separately for those two specific dependency types.

Reply:
Also note that this is initializing missing map entries from the default settings [here](https://github.com/stacklok/minder/pull/3392/files#diff-4747ed96b4116a79071ea012528f62f247e7f143e6d8e7f276739816cb54d163R179-R181).

Reply:
I'm actually fine with this config granularity @rdimitrov . The way I read it the action tells you the "what" to do with the PR and the "block" tells you which of the attributes to take into account when deciding whether to block. And I also think all of them should be on by default.

Reply:
Ah, thanks for flipping the setting! 😃 Yeah, my motivation was to keep vulncheck and trusty behaviour similar, but I agree on the functionality of this + we'll be refactoring that part so we might as well make it available for all PR related Minder actions not just Trusty.